trustmanager/keyfilestore.go

```diff
@@ -147,7 +310,44 @@ func (s *KeyMemoryStore) ExportKey(name string) ([]byte, error) {
 // ImportKey imports the private key in the encrypted bytes into the keystore
 // with the given key ID and alias.
 func (s *KeyMemoryStore) ImportKey(pemBytes []byte, alias string) error {
-	return importKey(s, s.Retriever, s.cachedKeys, alias, pemBytes)
+	err := importKey(s, s.Retriever, s.cachedKeys, alias, pemBytes)
+	if err != nil {
+		return err
+	}
+	keyID, keyInfo := getImportedKeyInfo(pemBytes, alias)
+	s.keyInfoMap[keyID] = keyInfo
+	return nil
+}
+
+func getImportedKeyInfo(pemBytes []byte, filename string) (string, KeyInfo) {
```

Comment:
Agreed with the rename.

After discussing IRL, it seems that we could benefit from removing `ImportRootKey` and refactoring the keystore `ImportKey` API to have a different signature: `ImportKey(pemBytes []byte, role, gun string)`.  This will make the code much easier to understand -- there's currently a lot of weirdness with importing keys.   For example: the `alias` existing argument to `ImportKey` can be a full filepath or just `"root"`